packages/accordion/src/AccordionContainer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+
+import { useAccordion } from './useAccordion';
+
+export function AccordionContainer({ children, render = children, ...props }) {
```

Comment:
nit picky but in all other containers I've named `props` as `options`.

Reply:
lol. I must have been looking at https://github.com/zendeskgarden/react-containers/blob/master/packages/breadcrumb/src/BreadcrumbContainer.js or https://github.com/zendeskgarden/react-containers/blob/master/packages/schedule/src/ScheduleContainer.js. Updated to match the majority 😂 .

Reply:
heh, I think I changed my mine after those ones. Will do an update to those to match all the others